composer.json

```diff
@@ -29,13 +29,14 @@
         "source": "https://github.com/wintercms/winter"
     },
     "require": {
-        "php": "^8.0.2",
+        "php": "^8.1",
```

Comment:
What's the PHP 8.1 feature that we're using?

Reply:
Interface const overriding (see: https://www.php.net/manual/en/language.oop5.interfaces.php), but we're not using it any more so could rollback that change